### ISSUE TITLE
mbp14,3: grub, visual glitch, missing arrows

### ISSUE THREAD

**Issue by aapo** [Dunedan/mbp-2016-linux#55]
MacBookPro14,3
Ubuntu: 18.04
Grub: 2.02

In Grub menu there are missing characters: up-arrow and down-arrow.

This is only cosmetic issue, but might relate #54 
![20180303_113558](https://user-images.githubusercontent.com/132666/36944371-71415848-1fa3-11e8-8bb0-4a24159b8c9a.jpg)

**Comment by aapo** [comment]
Ubuntu 18.04 tested in virtualbox works as expected.
![working_grub](https://user-images.githubusercontent.com/132666/36944407-2265d4e6-1fa4-11e8-8d94-e1e9da5f3e9c.png)

**Comment by aapo** [comment]
Context in code: grub2-2.02/grub-core/normal/menu_text.c:170
 
```
     msg_translated = grub_xasprintf (_("Use the %C and %C keys to select which "
					 "entry is highlighted."),
				       GRUB_UNICODE_UPARROW,
				       GRUB_UNICODE_DOWNARROW);
```

**Comment by Dunedan** [comment]
Works for me with Debian/unstable, but I also have a graphical grub menu enabled. I doubt that this is a hardware related problem.

**Comment by aapo** [comment]
Fixed in Grub: 2.02-2ubuntu8
